src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -1,7 +1,10 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
 
-import Cookie
+try:
+    import Cookie
+except ImportError:
+    import http.cookies as Cookie
```

Comment:
```suggestion
import http.cookies as Cookie
```

no need to be compatible with Python2.

Reply:
> no need to be compatible with Python2.
@tchaikov you sure? Nautilus was Py2-Py3 compatible if I'm not wrong (in fact Red Hat downstream Ceph supports both in CentOS 7).

Reply:
ahh, right. nautilus is the last release which supports python2. my bad.